lib/cisco_node_utils/bgp_neighbor_af.rb

```diff
@@ -452,6 +452,53 @@ def default_next_hop_third_party
     end
 
     # -----------------------
+    # <state> prefix-list <str> in
+    def prefix_list_in
+      str = config_get('bgp_neighbor_af', 'prefix_list_in', @get_args)
+      return default_prefix_list_in if str.nil?
+      str.shift.strip
+    end
+
+    def prefix_list_in=(str)
+      str.strip! unless str.nil?
+      if str == default_prefix_list_in
+        state = 'no'
+        # Current prefix-list name is required for removal
+        str = prefix_list_in
+        return if str.nil?
```

Comment:
Do you really need this return statement?  Does not look like prefix_list_in will ever return nil?